src/stores/AccountsStore.js

```diff
@@ -18,10 +18,9 @@
 
 import { Container } from 'unstated';
 import debounce from 'debounce';
-import { loadAccounts, saveAccount, deleteAccount } from '../util/db';
+import { loadAccounts, saveAccount } from '../util/db';
```

Comment:
The rule is: one feature (logical change) - one PR. Make a separate PR for these changes, please.